packages/app-mobile/components/NoteEditor/CodeMirrorLanguages.ts

```diff
@@ -0,0 +1,235 @@
+/**
```

Comment:
camelCase. And shouldn't that be called "syntaxHighlightingLanguages" (like what you export)